### ISSUE TITLE
Visualizing part segmentation using Viser

### ISSUE THREAD

**Issue by king1111sadjfoisja** [sharinka0715/semantic-gaussians#16]
Dear author:
I noticed that in `view_viser.py `,the 2D model used does not include VLPart.
![image](https://github.com/user-attachments/assets/86ee61c2-03a5-4625-866d-5362e6979c4d)
Could you please tell me how to visualize my part segmentation results?
Here is the rendered result of a scene from the MVImgNet dataset.
![image](https://github.com/user-attachments/assets/edd6a2c7-8c5a-45cb-8149-3bae19cc8474)
I want to know how to visualize my part segmentation results.
Any help is appreciated,thanks!

**Comment by HuGhcat-code** [comment]
same problem

**Comment by sharinka0715** [comment]
Hello,

Except from LSeg, all pretrained models match the ViT-L/14@336px CLIP embeddings, so you do not need to modify the code. Jus t use it.

If you cannot obtain proper part segmentation results, please check https://github.com/sharinka0715/semantic-gaussians/blob/ae531372ee37e4016030362397ddb1c64ad4c58d/model/vlpart_predictor.py#L25C9-L25C27 to ensure the part prompts is in the predefined classes.

**Comment by HuGhcat-code** [comment]
Hi,
I rendered a mug scene from the MVImagnet dataset, and after running view_viser.py, I encountered the visualization interface shown in the image. However, when I input text prompts and click 'apply text prompt,' nothing happens. How should I solve this issue?
![image](https://github.com/user-attachments/assets/404f7a52-f49e-45b4-9d64-92666b249127)
PS:the part prompt "handle"is in the predefined classes as follows:
![image](https://github.com/user-attachments/assets/f701db53-b29c-46a1-9677-e150cae75994)
